src/main/java/com/redhat/pantheon/servlet/AsciidocRenderingServlet.java

```diff
@@ -140,9 +140,11 @@ private Content generateHtml(SlingHttpServletRequest request, Resource resource)
             ob = ob.templateDir(dependencyProvider.getTemplateDir());
         }
 
+        long start = System.currentTimeMillis();
         c.html = getDependencyProvider().getAsciidoctor().convert(
                 c.asciidoc,
                 ob.get());
+        log.info("Rendering finished in {} ms.", System.currentTimeMillis() - start);
```

Comment:
I prefer .info here, since with the caching in place we shouldn't hit this all that often - I'd like to know when we do hit it so I can make sure that 1.) it's not unexpected, and 2.) we get a feel for how long it takes on average. (And by extension, if a request IS taking a long time, is it asciidoctor or something else?)

Reply:
:+1: